submanifests/optional.yaml

```diff
@@ -46,7 +46,7 @@ manifest:
       groups:
         - optional
     - name: tflite-micro
-      revision: 1a34dcab41e7e0e667db72d6a40999c1ec9c510c
+      revision: /pull/6/head
```

Comment:
need to remove leading slash /

Reply:
update to e1eecbee18aa8f7f0ad7e90f8b335564eb52c13f please

Reply:
based on the reviews and updates in https://github.com/zephyrproject-rtos/tflite-micro/pull/6,

the sha should be 48613f7ba1ffbda46ad771a77a35408f48f922e9 instead. (https://github.com/zephyrproject-rtos/tflite-micro’s `zephyr` branch should be updated accordingly)

Reply:
@ithinuel is correct, the original export commit had some duplicated files, please repush the last commit @nashif

Reply:
@JordanYates 

done